### PR TITLE
FEAT: Added Home page,Routing among pages

### PR DESCRIPTION
**Pull request by kotesh-arya**
-  #### Static Home page
  - #### Basic Routing
https://user-images.githubusercontent.com/69259490/160920755-c7ebab3a-4b57-4488-a73a-a6737a71f847.mp4

### CODE REVIEW

File: src/pages/HomePage/HomePageComponents/Banner/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import '../../../../Colours/colours.css';
```
Comment (TejasShekar) [inline src/pages/HomePage/HomePageComponents/Banner/Banner.jsx:2]: You can make use of absolute paths.
Read more about it from here. https://dev.to/theodorusclarence/clean-up-your-imports-using-absolute-imports-alias-on-react-apps-next-js-cra-and-react-vite-2160

File: src/pages/HomePage/HomePageComponents/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import "../../../../Colours/colours.css";
+import '../Footer/Footer.css'
+import { FaGithub,FaLinkedin,FaTwitter } from 'react-icons/fa';
```
Comment (TejasShekar) [inline src/pages/HomePage/HomePageComponents/Footer/Footer.jsx:4]: Nice. Didn't know this existed. Learnt something new.

**Review by TejasShekar — COMMENTED** [review]
Good job Kotesh. I have left a suggestion. Have a loot at it.

- Also provide a live preview link as I'm unable to play the video you've uploaded in the description.

File: src/pages/HomePage/HomePageComponents/Footer/Footer.jsx (hunk 2 above)
Comment (kotesh-arya) [reply src/pages/HomePage/HomePageComponents/Footer/Footer.jsx:4]: Thank You 🙂

File: src/pages/HomePage/HomePageComponents/CategoryCard/CategoryCard.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    width: 100%;
+    height: 300px;
+    object-fit: cover;
+    object-position: bottom;
+}
+
+.card {
+    width: 20rem;
+    margin: 0 1rem;
+}
```
Comment (ark2002) [inline src/pages/HomePage/HomePageComponents/CategoryCard/CategoryCard.css]: prefer using rems for anything above 10 px

File: src/pages/HomePage/HomePageComponents/NavBar/NavBar.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+.number-one {
+    background-color: var(--alert-primary);
+    height: 1.5rem;
+    width: 1.5rem;
+    border-radius: 50%;
+    position: relative;
+    left: -0.6rem;
+    margin-right: -1.3rem;
+    bottom: 0.8rem;
```
Comment (ark2002) [inline src/pages/HomePage/HomePageComponents/NavBar/NavBar.css]: dont use rems in 0. formats

**Review by ark2002 — APPROVED** [review]
some minor changes rest code looks clean and easy to read

File: src/pages/HomePage/HomePageComponents/CategoryCard/CategoryCard.css (hunk 3 above)
Comment (kotesh-arya) [reply src/pages/HomePage/HomePageComponents/CategoryCard/CategoryCard.css]: Sure...
Will implement it 👍️

File: src/pages/HomePage/HomePageComponents/NavBar/NavBar.css (hunk 4 above)
Comment (kotesh-arya) [reply src/pages/HomePage/HomePageComponents/NavBar/NavBar.css]: Sure, I will correct it and not repeat it again 👍️

**Review by TejasShekar — APPROVED** [review]

**Pull request merged**